common/lib/protocol-definitions/src/clients.ts

```diff
@@ -44,16 +44,6 @@ export interface ISignalClient {
     clientId: string;
 
     client: IClient;
-
-    /**
-     * Counts the number of signals sent by the client
-     */
-    clientConnectionNumber?: number;
-
-    /**
-     * Sequence number that indicates when the signal was created in relation to the delta stream
-     */
-    referenceSequenceNumber?: number;
```

Comment:
Was removing these intentional?

Reply:
yeah. this looks wrong to me. the problem is how the runtime handles signals and strips the envelope

Reply:
the op and signal workload should be symmetrical, and is op the refSeq is on the outermost envelope. ideally like ops, the loader layer also adds this to outgoing signals as well

Reply:
ahh. thought this was ISignalMessage, but that is below, so this looks good

Reply:
`initialClients` is `ISignalClient[]`. 
If this is removed, then the server cannot tell the connection/ref numbers for existing clients when a new client connects.

Reply:
we probably want to keep this if want clients from initial clients to also have their ref seq, and connectionnumber